ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/rewriter/RemoveBranch.scala

```diff
@@ -0,0 +1,39 @@
+package pl.touk.nussknacker.restmodel.displayedgraph.rewriter
+
+import pl.touk.nussknacker.engine.graph.node.NodeData
+import pl.touk.nussknacker.restmodel.displayedgraph.DisplayableProcess
+import pl.touk.nussknacker.restmodel.displayedgraph.displayablenode.Edge
+
+class RemoveBranch(process: DisplayableProcess, branchRoot: NodeData) {
```

Comment:
The same comment to process field as in `RemoveAndBypassNode`. Without this it is hard to say how many times you are iterating through the edges/nodes of process.